### PR TITLE
refactor[yaml]: Modified the application specific pipeline for cassandra

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: Aman Gupta aman.gupta@mayadata.io

1. Resolved typos in stages.
2. modified sed commands.
3. Used openebs-cstor-disk in provision of cassandra.

### CODE REVIEW

File: Openshift-EE/pipelines/Cassandra/stages/4-chaos/OI4T-cstor-target-network-delay/cstor-target-network-delay [hunk 1]
```diff
@@ -82,7 +82,7 @@ testResult=$(kubectl get litmusresult ${test_name} --no-headers -o custom-column
 # Update the e2e cr once the job is completed
 bash Openshift-EE/utils/e2e-cr jobname:cstor-target-network-delay jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" test_result:$testResult
 
-python3 Openshift-EE/utils/result/result_update.py $job_id $case_id 4-chaos "Verify application availablity after inducing delay in cstor target." $testResult $pipeline_id "$current_time" $commit_id $gitToken
+python3 Openshift-EE/utils/result/result_update.py $job_id $case_id 4-chaos "Verify application availablity after inducing delay in cstor target network." $testResult $pipeline_id "$current_time" $commit_id $gitToken
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/Cassandra/stages/4-chaos/OI4T-cstor-target-network-delay/cstor-target-network-delay]: @w3aman `inducing network delay on cstor target.`

File: Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay [hunk 2]
```diff
@@ -87,7 +87,7 @@ current_time=$(eval $time)
 # Update the e2e cr once the job is completed
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay]: @w3aman can you include the sed command to append the run_id.

Comment (w3aman) [reply Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay]: Done

File: Openshift-EE/pipelines/Cassandra/stages/4-chaos/OI4T-cstor-target-network-delay/cstor-target-network-delay (hunk 1 above)
Comment (w3aman) [reply Openshift-EE/pipelines/Cassandra/stages/4-chaos/OI4T-cstor-target-network-delay/cstor-target-network-delay]: Changed

**Pull request merged**